metrics/error-report/Bugsnag.js

```diff
@@ -51,43 +48,46 @@ function initClient(sourceId) {
   // This also prevents unnecessary errors sent to Bugsnag during development phase.
   const apiKeyRegex = /{{.+}}/;
   if (API_KEY.match(apiKeyRegex) !== null) return;
+  const host = window.location.hostname;
+  const devHosts = ['localhost', '127.0.0.1'];
```

Comment:
Even "[::1]" is also an alias for localhost. Please test and add it as well.
Example: https://app.bugsnag.com/rudderlabs/sdk-error-reporter/errors/62cc51f4c62e8d00072e872c?filters[event.since]=all&filters[search]=matomo&event_id=62cc54bd009566d3678f0000

https://en.wikipedia.org/wiki/Localhost

Reply:
> Example: [https://app.bugsnag.com/rudderlabs/sdk-error-reporter/errors/62cc51f4c62e8d00072e872c?filters[event.since]=all&filters[search]=matomo&event_id=62cc54bd009566d3678f0000](https://app.bugsnag.com/rudderlabs/sdk-error-reporter/errors/62cc51f4c62e8d00072e872c?filters%5Bevent.since%5D=all&filters%5Bsearch%5D=matomo&event_id=62cc54bd009566d3678f0000)

The host name in above issue is '127.0.0.1'.
I am adding '::1' but not sure how to test it.

Reply:
Sorry. Tagged the wrong URL.
The correct one,
https://app.bugsnag.com/rudderlabs/sdk-error-reporter/errors/62ccfb152d63fc0008f241c5?event_id=62ccfb1500956547b7400000&i=wh&m=nw.

Reply:
Thanks for the ref but couldn’t test this particular "::1" hostname.

Reply:
The hostname should be `[::1]` and not `::1`.

Reply:
Addressed.